lib/model/folder_sendrecv.go

```diff
@@ -592,8 +592,7 @@ func (f *sendReceiveFolder) handleDir(file protocol.FileInfo, snap *db.Snapshot,
 		// Check that it is what we have in the database.
 		curFile, hasCurFile := snap.Get(protocol.LocalDeviceID, file.Name)
 		if err := f.scanIfItemChanged(file.Name, info, curFile, hasCurFile, scanChan); err != nil {
-			err = fmt.Errorf("handling dir: %w", err)
-			f.newPullError(file.Name, err)
+			f.newPullError(file.Name, fmt.Errorf("handling dir: %w", err))
```

Comment:
Hmm, maybe we're not entirely consistent here, but I think praxis is for the prefix to be that of the executing function. That is, if I call `os.Open("foo")` it will return `open: foo: no such file`, so I took `handling dir` as we are in `handleDir`... The outer context will be why we're handling the dir, maybe.

Reply:
I always understood those contexts to differentiate multiple errors returned from the same function, so one can know from which operation inside the function came from. Whichever it is we should conform to one, otherwise we end up with "handle dir: handling dir: err" at some point. Anyway, not a concern for this PR.

Reply:
I take it upon myself to harmonize errors sanely at some point.